lib/core/macros.nim

```diff
@@ -1319,6 +1319,9 @@ proc customPragmaNode(n: NimNode): NimNode =
           if identDefs.kind == nnkRecCase:
             identDefsStack.add(identDefs[0])
             for i in 1..<identDefs.len:
+              # if it is and empty 'else' branch, skip
+              if identDefs[i].kind == nnkElse and
```

Comment:
not sure if it is handled, non else branches can be empty as well.

Reply:
true, i will fix it